app/services/symphony_client.rb

```diff
@@ -43,9 +43,6 @@ def patron_info(patron_key:, session_token:, item_details: {})
                                        ].join(',')
                                      })
     JSON.parse(response.body)
-  rescue HTTP::TimeoutError => e
-    # Allow execution to proceed. Error response handled by validating Patron creation downstream.
-    puts "Rescued: #{e.inspect}"
```

Comment:
no need for this anymore?

Reply:
this was _needed_ for the prior work that was reversed... I don't know that we _need_ it anymore but maybe keeping it would be a good idea. Lemme think about that.

Reply:
any thoughts on this? not keeping it?

Reply:
nope, not keeping it. probably would have been better to do rescue HTTP::Error as a blanket catch, but regardless, no. a rescue here prevents the desired action from happening - which is a retry. If it times out after a retry it'll just fail and after a time the javascript will prompt the user that a bad thing happened. I was looking for a way to sort of detect that we were on the last attempt of the job and then rescue there and return back an error, but I couldn't find a way to do that in ActiveJob. Sidekiq has a method, but overriding the time between tries formula seemed too difficult and perhaps [even not recommended](https://github.com/mperham/sidekiq/issues/4281#issuecomment-543815196)

Reply:
I agree let's not keep it